tests/core/gadget-kernel-refs-update-pc/task.yaml

```diff
@@ -109,7 +113,8 @@ restore: |
     #      restore requires a reboot :/
 
     # remove SNAPD_SHUTDOWN_DELAY again
-    sed -i 's/SNAPD_SHUTDOWN_DELAY=1//g' /etc/systemd/system/snapd.service.d/local.conf
+    rm "$CONF_OVERRIDE"
```

Comment:
we should either use ´tests.cleanup defer' here or move to the restore the tests.cleanup steps in order to
keep consistency.